R/radae.R

```diff
@@ -138,7 +139,7 @@ radae <- function(ADSL,
       AETOXGR %in% c(4, 5) ~ "SEVERE"
     ))
 
-  ADAE <- var_relabel(
+  ADAE <- formatters::var_relabel(
```

Comment:
```suggestion
  ADAE <- var_relabel(
```
okok do not want automation to get mad at this ahah. @edelarua it is in the imports like lubridate, why do you think is it complaining? This modification it is not necessary at all btw

Reply:
@Melkiades `formatters` is not imported for this package, the `var_relabel` function was added directly to `random.cdisc.data` instead.